benchmarking/driver/benchmark_driver.py

```diff
@@ -68,14 +82,19 @@ def _runOnePass(info, benchmark, framework, platform):
         "At this moment, only one test exists in the benchmark"
     output, treatment_files = \
         framework.runBenchmark(info, benchmark, platform)
+
+    data = None
     test = benchmark["tests"][0]
     if treatment_files and test["metric"] == "error":
         golden_files = test["output_files"]
         data = _processErrorData(treatment_files, golden_files)
     elif test["metric"] == "delay":
         data = _processDelayData(output)
+    elif test["metric"] == "generic":
+        data = output
```

Comment:
Just FYI, at some point when we have more metrics, we need to refactor this code so that all metrics have the same API, using some derived class from a base class and avoid this condition. 

That will clean this up.